runtime/lua/vim/treesitter/query.lua

```diff
@@ -132,25 +135,38 @@ function M.list_predicates()
   return vim.tbl_keys(predicate_handlers)
 end
 
+local function xor(x, y)
+  return (x or y) and not (x and y)
+end
```

Comment:
Maybe we can have that as a fallback, and try `bit` as default ?

Reply:
no, we should just fix so that `bit` is always available (plugins also wants this). I just wanted to unblock tree-sitter progress.